evalml/model_understanding/prediction_explanations/_user_interface.py

```diff
@@ -146,63 +194,86 @@ def make_dataframe(self, shap_values, normalized_values, pipeline_features, top_
-                          convert_numeric_to_string=False)
+    def make_dict(self, aggregated_shap_values, aggregated_normalized_values,
+                  shap_values, normalized_values, pipeline_features, original_features):
+        rows = _make_rows(aggregated_shap_values, aggregated_normalized_values, pipeline_features, original_features,
+                          self.top_k, self.include_shap_values, convert_numeric_to_string=False)
         json_rows = _rows_to_dict(rows)
+        drill_down = self.make_drill_down_dict(self.provenance, shap_values, normalized_values,
+                                               pipeline_features, original_features, self.include_shap_values)
         json_rows["class_name"] = None
+        json_rows["drill_down"] = drill_down
         return {"explanations": [json_rows]}
 
 
 class _BinarySHAPTable(_TableMaker):
     """Makes a SHAP table explaining a prediction for a binary classification problem."""
 
-    def __init__(self, class_names):
+    def __init__(self, top_k, include_shap_values, class_names, provenance):
+        super().__init__(top_k, include_shap_values, provenance)
         self.class_names = class_names
 
-    def make_text(self, shap_values, normalized_values, pipeline_features, top_k, include_shap_values=False):
+    def make_text(self, aggregated_shap_values, aggregated_normalized_values,
+                  shap_values, normalized_values, pipeline_features, original_features):
         # The SHAP algorithm will return a two-element list for binary problems.
         # By convention, we display the explanation for the dominant class.
-        return _make_text_table(shap_values[1], normalized_values[1], pipeline_features, top_k, include_shap_values)
+        return _make_text_table(aggregated_shap_values[1], aggregated_normalized_values[1],
+                                pipeline_features, original_features, self.top_k, self.include_shap_values)
 
-    def make_dict(self, shap_values, normalized_values, pipeline_features, top_k, include_shap_values=False):
-        rows = _make_rows(shap_values[1], normalized_values[1], pipeline_features, top_k, include_shap_values,
-                          convert_numeric_to_string=False)
+    def make_dict(self, aggregated_shap_values, aggregated_normalized_values,
+                  shap_values, normalized_values, pipeline_features, original_features):
+        rows = _make_rows(aggregated_shap_values[1], aggregated_normalized_values[1], pipeline_features, original_features,
+                          self.top_k, self.include_shap_values, convert_numeric_to_string=False)
         json_rows = _rows_to_dict(rows)
+        drill_down = self.make_drill_down_dict(self.provenance, shap_values[1], normalized_values[1],
+                                               pipeline_features, original_features, self.include_shap_values)
+        json_rows["drill_down"] = drill_down
```

Comment:
Is "json_rows" a carryover copy pasta?  It's kinda weird in make_dict()

Reply:
Changed the name to `dict_rows`!